### PR TITLE
Milestone 3: Users and Posts

### PR DESCRIPTION
**Pull request by emmanuelkamala**
In this Milestone, I have created a user and post controllers, models and views
- I have used devise for user authentication
- I have added tests using rspec for the user and post

### CODE REVIEW

File: app/controllers/posts_controller.rb [hunk 1]
```diff
@@ -0,0 +1,61 @@
+class PostsController < ApplicationController
+
+
+    before_action :set_post, only: [:show, :edit, :update, :delete, :destroy]
+    before_action :authenticate_user!
+    
+    def index
+        #@posts = Post.all.order("created_at DESC")
```
Comment (ebukaume) [inline app/controllers/posts_controller.rb]: Commented code should never be committed. You seem to have forgotten this one here. Please remove!

**Comment by emmanuelkamala** [comment]
- [ ] I have edited my Users controller to list all users in the app.
- [ ] Also, I have changed my Posts controller to display all posts from users in the timeline

**Comment by emmanuelkamala** [comment]
Hi, I have made the requested changes:

1. I have filled the User's index.html.erb page with content
2. Changed the link on Mambo, when it is clicked when a user is signed in, it takes the user back to the list of all posts

File: .rubocop.yml [hunk 2]
```diff
@@ -0,0 +1,41 @@
+AllCops:
```
Comment (mcrd25) [inline .rubocop.yml]: [OPTIONAL]
While this configuration is fine, it is better to use the Ruby On Rails Specific configuration for the curriculum found here: https://github.com/microverseinc/linters-config/tree/master/ror

Comment (emmanuelkamala) [reply .rubocop.yml]: Thank you for the review Maya, will let you know when I get stuck.

**Pull request merged**